### DISCUSSION TITLE
Make the new tutorials notebooks directory as jupyter default

### DISCUSSION THREAD

**Discussion by ammarsani-my** [azavea/raster-vision#1593] [Ideas]
Hi @AdeelH @lewfish.

I just thought maybe we can make default docker volume mount path when running `docker/run --jupyter[-lab]` to your newly prepared tutorial notebooks path in the repository. Hence, this might probably make easier for new user like me to navigate thru them by just adding this argument to the command.

Just an idea! Thanks.

**Comment by AdeelH** [comment]
Thanks for the suggestion! This makes sense to me.

**Comment by lewfish** [comment]
This would be a good first contribution to the library if you want to make a PR.

**Reply by ammarsani-my** [reply]
Sure! Will do it soonest possible.

**Comment by ammarsani-my** [comment]
Hi. I just create a pull request on this idea at https://github.com/azavea/raster-vision/pull/1595.
